math/src/main/java/smile/math/Math.java

```diff
@@ -4411,7 +4411,7 @@ public static double min(DifferentiableMultivariateFunction func, int m, double[
         // Current function value.
         double f = func.f(x, g);
 
-        logger.info(String.format("L-BFGS: initial function value: %.5g", f));
+        logger.debug(String.format("L-BFGS: initial function value: %.5g", f));
```

Comment:
Why `debug`? I prefer see the messages to asses if the algorithm is converging.

Reply:
ok. I will revert this change.